src/components/PlanterDetail.js

```diff
@@ -195,11 +194,31 @@ const PlanterDetail = (props) => {
               </Typography>
             </Grid>
             <Divider />
+            <Grid container direction="column" className={classes.box}>
+              <Typography variant="subtitle1">Country</Typography>
+              <Typography variant="body1">
+                {(planterRegistrations &&
+                  planterRegistrations
+                    .map((item) => item.country)
+                    .filter(
+                      (country, i, arr) =>
+                        country && arr.indexOf(country) === i,
+                    )
+                    .join(', ')) ||
+                  '---'}
+              </Typography>
+            </Grid>
+            <Divider />
             <Grid container direction="column" className={classes.box}>
               <Typography variant="subtitle1">Registered</Typography>
               <Typography variant="body1">
-                {(planterRegistration &&
-                  getDateTimeStringLocale(planterRegistration.createdAt)) ||
+                {(planterRegistrations &&
+                  planterRegistrations.length > 0 &&
+                  getDateTimeStringLocale(
+                    planterRegistrations.sort((a, b) =>
```

Comment:
It would be simpler to do the sort when the registrations are loaded, rather than on each render.